framework/application/modules/Session/Session.php

```diff
@@ -208,10 +216,12 @@ public function getSessname()
 	 *
 	 * Checks to see if cookies are enabled. Returns TRUE if they are, FALSE if they aren't.
 	 *
+	 * @param		$js			Tell Javascript to check for cookies. Only works in conjunction with Document::addJSErrorBox().
+	 *									Default is FALSE. (This property is still here to support backward compatibility in calls)
 	 * @access public
 	 * @return bool
 	 */
-	public function checkCookies()
+	public function checkCookies($js=FALSE)
```

Comment:
It needs to stay there for backward compatibility. If we check through the code base and ensure that no script is calling that method with a param, we can remove it.